src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -234,11 +234,11 @@
   <data name="GenerateLocal" xml:space="preserve">
     <value>Generate local '{0}'</value>
   </data>
-  <data name="GenerateForInNewFile" xml:space="preserve">
-    <value>Generate {0} for '{1}' in '{2}' (in new file)</value>
+  <data name="Generate_0_1_in_new_file" xml:space="preserve">
```

Comment:
These underscores and numbers make my eyes bleed.  Is the IntelliSense not enough to visualize the resultant string?  E.g., the generated doc-comment that's of the form "Returns a string similar to Generate {0} for ..."